include/ignition/gazebo/gui/GuiRunner.hh

```diff
@@ -48,10 +48,6 @@ class IGNITION_GAZEBO_VISIBLE GuiRunner : public QObject
   /// \brief Destructor
   public: ~GuiRunner() override;
 
-  /// \brief Callback when a plugin has been added.
-  /// \param[in] _objectName Plugin's object name.
-  public slots: void OnPluginAdded(const QString &_objectName);
```

Comment:
Ahh I forgot that this header is installed, let's leave the function in, but without doing anything.

Reply:
9ecc40977235ea31eb9ca1df3f73b7605c685a89